doc/sphinx/biblio.bib

```diff
@@ -644,3 +644,15 @@ @inproceedings{ConchonFilliatre07wml
   x-support = {actes_aux},
   x-cle-support = {ML}
 }
+
+@phdthesis{Zimmermann19,
+  author       = {Th{\'{e}}o Zimmermann},
+  title        = {Challenges in the collaborative evolution of a proof language and
+                  its ecosystem.},
+  school       = {Paris Diderot University, France},
```

Comment:
FWIW, the university name at the time was "Université de Paris", now officially "Université Paris Cité".

Reply:
Congrats for managing to graduate in the short time interval that name survived!